electronics_lib/JLC_Resistor.py

```diff
@@ -0,0 +1,87 @@
+class JLCPCB_ResistorTable(ESeriesResistor):
+  RESISTANCE = PartsTableColumn(Range)  
+  POWER_DISSIPATION = PartsTableColumn(Range)  
+  PRICE = PartsTableColumn(float)
+  FOOTPRINT = PartsTableColumn(str)
+    
+  @classmethod
+  def _generate_table(cls) -> PartsTable:
+    RESISTOR_MATCHES = {
+    'resistance': "\s?(\d+(\.\d*)?[GMkmunp]?)?[\u03A9]\s",
```

Comment:
style: additional indent for dict contents. I generally follow IntelliJ's automatic styling, but with indents set to two spaces instead of the default four.

(also more indents needed below, but I won't flag every instance)